README.md

```diff
@@ -45,7 +45,7 @@ Here are two Spacebars templates from an example app called "Leaderboard" which
 </template>
 ```
 
-The template tags `{{name}}` and `{{score}}` refer to properties of the data context (the current player), while `players` and `selected` refer to helper functions.  Helper functions and event handlers are defined in JavaScript:
+The template tags `{{name}}` and `{{score}}` refer to properties of the data context (the current player), while `players` and `selected` refer to helper functions.
```

Comment:
Can you in both cases not use `{{` and `}}` around? Because it is not consistent. For example, it is `{{selected}}` in the example.